app/auth/models.py

```diff
@@ -0,0 +1,10 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
```

Comment:
Add a single line above `class User(db.Model)` to be consistent with pycodestyle.